audit/audit.go

```diff
@@ -117,3 +120,40 @@ func (a *Audit) onLoggerError(err error) {
 		a.OnError(err)
 	}
 }
+
+// sortAuditFields sorts the context fields of an audit record such that some fields
+// are prepended in order, some are appended in order, and the rest are sorted alphabetically.
+// This is done to make reading the records easier since common fields will appear in the same order.
+func sortAuditFields(fields logr.Fields) []format.ContextField {
```

Comment:
What about a unit test for this?

Reply:
Great, thanks @jwilander .  Unit test added here https://github.com/mattermost/mattermost-server/pull/14062/commits/b696b154297876c4da18c085f78e77ab78792def